packages/traceability-schemas/services/contexts.js

```diff
@@ -1,204 +1,13 @@
 const fs = require('fs');
 const path = require('path');
-const cred = require('@transmute/credentials-context');
```

Comment:
I bundled all the contexts we rely on in the contexts directory, and removed this vendor dep.